### DISCUSSION TITLE
Aeotec NanoMote Quad (ZWA003) in secure mode

### DISCUSSION THREAD

**Discussion by kpanchen** [zwave-js/zwave-js#6491] [Request Support / Investigate Issue]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening? - Device can be included in both secure (S2) and unsecure mode, both inclusion successful but with secure mode after device goes to sleep it will never communicate with controller again. Unsecure works fine.

What did you expect to happen instead?

Both mode works the same way.

Anything else we should know?

All the attempts are next to the controller, this is the second problematic with security legacy Aeotec device, just want to make sure problem is not on my side. Side note - old controller is still running as devices being moved to a new one.

[zwavejs_2023-11-03.log](https://github.com/zwave-js/node-zwave-js/files/13248436/zwavejs_2023-11-03.log)

### Software versions

zwave-js-ui: 9.3.1
zwave-js: 12.2.3

Stand alone

### Device information

Manufacturer: Aeotec
Model name: Nanomote Quad
Node ID: 54

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[z-ui_2023-11-03.log](https://github.com/zwave-js/node-zwave-js/files/13248429/z-ui_2023-11-03.log)

**Comment by AlCalzone** [comment]
Not sure if this is related, but it seems like your device doesn't actually have parameter 41. We have it in our [configuration DB](https://devices.zwave-js.io/?jumpTo=0x0371:0x0002:0x0003:0.0), but [Aeotec's manual](https://aeotec.freshdesk.com/support/solutions/articles/6000184696-nanomote-quad-user-guide-) does not list it (down at the bottom).

I have to agree though, there's nothing in that log that looks like any `Central Scene` commands are received when the device is included with S2. Might be worth asking Aeotec about - not sure though if they'll do anything about it at this point.

If not, we can add a warning to the device DB to not include it securely.

**Reply by kpanchen** [reply]
I don't think Parameter 41 is related, I've noticed that you have it, it was in the old PDF documentation but removed for a new version:

![image](https://github.com/zwave-js/node-zwave-js/assets/43836896/ff4cd474-8d1f-465e-9e62-040a6aff9452)

When device is responding trying to set this parameter just gives an error that such one is not available. You might remove it or at least make a note that it is not available in firmware v1.2.

As for secure inclusion I agree I don't see any communication at all after device goes to sleep. When device is just paired it is blinking green indicating that it is awake and I can communicate with it, send parameters and read them just fine. As soon as I put it to sleep it will stop communicating and not even sending Wake-up notification when I put it back to awake mode. No communication to the level of not sending "Reset locally" when I do a hard reset of the device and re-include it again.
I don't think Aeotec will do anything about old device, definitely no new firmware. Might as well add a note to use it in un-secure mode.

Yet again big thank you for quick and detailed response!!! Really appreciate it.
K.
